sycl/source/detail/config.hpp

```diff
@@ -163,6 +165,118 @@ template <> class SYCLConfig<SYCL_PI_TRACE> {
   }
 };
 
+template <> class SYCLConfig<SYCL_DEVICE_TRIPLE> {
+  using BaseT = SYCLConfigBase<SYCL_DEVICE_TRIPLE>;
+
+public:
+  static std::vector<triple> *get() {
+    static bool Initialized = false;
+    static std::string String;
```

Comment:
Please, give the variable more meaningful name.

Reply:
It seems there is not reason for this var to be static.
Also, it seems this variable is not necessary at all, it just adds overhead on copying the string.